### DISCUSSION TITLE
Not able to hear sound when Multi-Output Device is selected (macOS)

### DISCUSSION THREAD

**Discussion by spaja** [ExistentialAudio/BlackHole#757] [Q&A]
Hi, I followed the instructions but when I select Multi-Output Device I do not hear any music from youtube on my airpods that are connected. Here is my setup:
<img width="968" alt="image" src="https://github.com/ExistentialAudio/BlackHole/assets/2871653/34bdb95f-e0b8-482d-9db4-8af23373007a">
As you can see here my multi output device is not listed so I just selected Black Hole:
<img width="814" alt="image" src="https://github.com/ExistentialAudio/BlackHole/assets/2871653/d920af8c-149f-4513-815d-db784852d9b1">

What am I doing wrong?

Thanks

**Comment by devinroth** [comment]
Nothing wrong. Just bugs that Apple won’t fix. AirPods don’t work well with multi outputs.

You can try using your laptop’s built in mic instead and see if that helps.

Devin
